README.md

```diff
@@ -33,18 +33,19 @@ For either platform, you can alternatively pass down a child element that will b
+| ----------------------- | -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- | ------------- |
+| onValueChange           | function       | Callback which returns `value, index`                                                                                                                                                                                                                                                                                             | Y         | Both          |
+| items                   | array          | _ The items for the component to render. Each item should be in the following format:<br>`{label: 'Orange',value: 'orange',key: 'orange', color: 'orange'}`<br>_ The label and the value are required, but the key will be based upon the label if it isn't included<br>\* The value can be any data type. The color is optional. | Y         | Both          |
+| placeholder             | object         | _ An override for the default placeholder object with a label of `Select an item...` and a value of `null`<br>_ An empty object can be used if you'd like to disable the placeholder entirely                                                                                                                                     | N         | Both          |
+| disabled                | boolean        | Disables interaction with the component                                                                                                                                                                                                                                                                                           | N         | Both          |
+| value                   | any            | Will attempt to locate a matching value from the `items` array by checking each item's `value` property. If found, it will update the component to show that item as selected. If the value is not found, it will default to the first item.                                                                                      | N         | Both          |
+| itemKey                 | string, number | Will attempt to locate a matching value from the `items` array by checking each item's `key` property. If found, it will update the component to show that item as selected. If the key is not found, it will default to the first item. **If both `value` and `itemKey` props are provided, `itemKey` supercedes.**              | N         | Both          |
```

Comment:
This description is kind of confusing because the first sentence contradicts the last -- it reads as if the method will try to find a matching value, then check the key property and return the valid item based on the key... but it's actually finding a matching key, and returning the valid item based on the key before it looks at the value.